apps/dashboard/src/components/file-upload-area.tsx

```
@@ -0,0 +1,76 @@
+
+    const handleFileInput = (e: React.ChangeEvent<HTMLInputElement>) => {
+        const selectedFile = e.target.files?.[0];
+        if (selectedFile) {
+            onFileSelect(selectedFile);
+        }
+    };
+
+    return (
+        <Card className="w-full max-w-2xl">
+            <CardContent className="p-0">
+                {/** biome-ignore lint/a11y/noStaticElementInteractions: drag and drop functionality requires these interactions */}
```

Biome (1.9.4)

[error] failed to parse category "lint/a11y/noStaticElementInteractions"



(suppressions/parse)